### PR TITLE
docs(networking): convert non-mesh traffic doc to use targetref policies

### PR DESCRIPTION
**Pull request by slonka**
convert non-mesh traffic doc to use targetref policies since 2.6

xref: https://github.com/kumahq/kuma-website/issues/1599

Did you sign your commit? [Instructions](https://github.com/kumahq/.github/blob/main/CONTRIBUTING.md#sign-your-commits) yes

Have you read [Contributing guidelines](https://github.com/kumahq/.github/blob/main/CONTRIBUTING.md)? yes

2.5.0: https://deploy-preview-1615--kuma.netlify.app/docs/2.5.x/networking/non-mesh-traffic/#non-mesh-traffic
dev: https://deploy-preview-1615--kuma.netlify.app/docs/dev/networking/non-mesh-traffic/#non-mesh-traffic

### CODE REVIEW

File: app/_src/networking/non-mesh-traffic.md [hunk 1]
```diff
@@ -70,6 +70,7 @@ maxRetries: 3
 
 Proxy Template to change the defaults:
```
Comment (lahabana) [inline app/_src/networking/non-mesh-traffic.md:71]: This should be in the if_version block

File: app/_src/networking/non-mesh-traffic.md [hunk 2]
```diff
@@ -147,6 +179,7 @@ tcp:
...
```
Comment (lahabana) [inline app/_src/networking/non-mesh-traffic.md:180]: same

File: app/_src/networking/non-mesh-traffic.md [hunk 3]
```diff
@@ -213,3 +246,35 @@ conf:
 ```
 {% endtab %}
 {% endtabs %}
+{% endif_version %}
+
```
Comment (bartsmykla) [inline app/_src/networking/non-mesh-traffic.md:250]:
```suggestion
```

File: app/_src/networking/non-mesh-traffic.md [hunk 4]
```diff
@@ -134,6 +135,37 @@ conf:
...
```
Comment (bartsmykla) [inline app/_src/networking/non-mesh-traffic.md:139]: between `if_version` blocks shouldn't be emply line
```suggestion
```

**Comment by slonka** [comment]
policy_yaml seems to incorrectly render this part:

image

vs

image

😢

**Review by lahabana — APPROVED** [review]

**Pull request merged**